sycl/doc/extensions/Reduction/Reduction_status.md

```diff
@@ -0,0 +1,189 @@
+# Reductions: current implementation and known problems/TODOs
+
+**NOTE**: This document is a quick draft. It is written to help developers of SYCL headers/library to understand the current status, currently used algorithms and known problems.
+
+
+
+# Reduction specifications
+
+There are 2 specifications of the reduction feature and both are still actual:
+
+* `sycl::ext::oneapi::reduction` is described in [this document](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/Reduction/Reduction.md). This extension was created as part of a pathfinding/prototyping work before it was added to SYCL 2020 standard.
+
+* `sycl::reduction` is described in [SYCL 2020 standard](https://www.khronos.org/registry/SYCL/specs/sycl-2020/html/sycl-2020.html#sec:reduction).
+
+These two specifications for reduction are pretty similar. The implementation of `sycl::reduction` is based on (basically re-uses) the implementation of `sycl::ext::oneapi::reduction`. 
+
+There are non-critical differences in API to create the reduction object. `sycl::reduction` accepts either `sycl::buffer` or `usm memory` and optional property `property::reduction::initialize_to_identity` as parameter to create a reduction, while `sycl::ext::oneapi::reduction` accepts `sycl::accessor` that has `access::mode` equal to either `read_write` (which corresponds to SYCL 2020 reduction initialized without `property::reduction::initialize_to_identity`) or `discard_write`(corresponds to case when `property::reduction::initialize_to_identity` is used).
+
+---
+---
+# Implementation details: `reduction` in `parallel_for()` accepting `nd_range`
+
+## Reduction inside 1 work-group - the main building block for reduction
+
+The reduction algorithm for 1 basic block depends currently on combination/availability of 2 features:
```

Comment:
```suggestion
The reduction algorithm for 1 work-group depends currently on combination/availability of 2 features:
```

Reply:
Misprint. Thank you. Fixed as suggested: 55c3d7537ae5a3ac89729ee4fb88cad41873ab5d